src/transport/SessionManager.h

```diff
@@ -210,8 +210,8 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate
     }
 
     // TODO: implements group sessions
-    Optional<SessionHandle> CreateGroupSession(GroupId group) { return mGroupSessions.AllocEntry(group, kUndefinedFabricIndex); }
-    Optional<SessionHandle> FindGroupSession(GroupId group) { return mGroupSessions.FindEntry(group, kUndefinedFabricIndex); }
+    Optional<SessionHandle> CreateGroupSession(GroupId group) { return mGroupSessions.AllocEntry(group, 1); }
+    Optional<SessionHandle> FindGroupSession(GroupId group) { return mGroupSessions.FindEntry(group, 1); }
```

Comment:
Shouldn't this be taking the fabric index as input?

Reply:
done